### ISSUE TITLE
cant connect with telegram desktop

### ISSUE THREAD

**Issue by erfantkerfan** [alexbers/mtprotoproxy#93]
I rented a new server in France (OVH) and used the stable branch but with my home network I cant connect with telegram desktop but I can use it with the same network with my android phone.
and I can use other mtprotos with my telegram desktop.(latest version of telegram desktop windows)
also I have ssh access with same network.
what other details should I provide?

**Comment by alexbers** [comment]
hello, I can try to debug an issue if you send the proxy address. I am @bay3255 in telegram

**Comment by m8603164** [comment]
I have the same problem.
Does not work on the desktop telegram and X telegram. But in the main telegram, it works correctly.
However, if the VPN is turned on, then the problem will be resolved!
All this while the SSH connection is without any problems.

**Comment by alexbers** [comment]
it the problem reproducible with the official proxy?

**Comment by m8603164** [comment]
Sorry, I did not understand your meaning.
This problem exists with your source and with the original source introduced by the telegram.
This problem is already present in the Hetzner servers, but it does not exist in DigitalOcean or Vultr!
But there's another problem with the DigitalOcean or Vultr!, which is the problem of downloading images and videos!

**Comment by alexbers** [comment]
It can be because of blocking. If you send my (better using the telegram) an ip and port the proxy you have problems with, I can set up one of my servers to redirect incoming traffic to the proxy to check.

**Comment by m8603164** [comment]
Yes, when it resolves the problem with VPN, that is, some sort of "ISP" detects traffic.
If necessary, I will send it to your telegram ID.

**Comment by alexbers** [comment]
@bay3255
